src/main/resources/application.yml

```diff
@@ -40,6 +40,7 @@ jobs:
     name: "BatchPrintRequests"
     cron: "0 0/15 * * * *" # every 15 minutes, starting on the hour - see analysis and recommendations in EIP1-2515
     batch-size: 50
+    max-un-batched-records: 5000
```

Comment:
5000, as per the recommendation in EIP1-2885 .... perhaps I should put a comment after it referencing the ticket number so someone can understand the rationale for this number at some later date 🤔

Reply:
Done